src/test/scala/cromwell/engine/db/slick/SlickDataAccessSpec.scala

```diff
@@ -38,6 +38,8 @@ class SlickDataAccessSpec extends FlatSpec with Matchers with ScalaFutures {
   object UnknownBackend extends Backend {
     override def adjustInputPaths(call: Call, inputs: CallInputs) = Map.empty
     override def adjustOutputPaths(call: Call, outputs: CallOutputs): CallOutputs = outputs
+    override def stdout(workflowId: WorkflowId, callFqn: FullyQualifiedName): WdlFile = ???
+    override def stderr(workflowId: WorkflowId, callFqn: FullyQualifiedName): WdlFile = ???
```

Comment:
I believe this class exists solely to test the handling of unknown backends and thus these methods aren't used